projects/golang/go/1.18/rpmbuild/SPECS/golang.spec

```diff
@@ -0,0 +1,1355 @@
+%global gohostarch  amd64
+%endif
+%ifarch %{ix86}
+%global gohostarch  386
+%endif
+%ifarch %{arm}
+%global gohostarch  arm
+%endif
+%ifarch aarch64
+%global gohostarch  arm64
+%endif
+%ifarch ppc64
+%global gohostarch  ppc64
+%endif
+%ifarch ppc64le
+%global gohostarch  ppc64le
+%endif
+%ifarch s390x
+%global gohostarch  s390x
+%endif
+
+# Comment out go_prerelease and go_patch as needed
+%global go_api 1.18
+#global go_prerelease rc1
+%global go_patch 3
+
+%global go_version %{go_api}%{?go_patch:.%{go_patch}}%{?go_prerelease:~%{go_prerelease}}
+%global go_source %{go_api}%{?go_patch:.%{go_patch}}%{?go_prerelease}
+ 
+# For rpmdev-bumpspec and releng automation
+%global baserelease 1
+ 
+Name:           golang
+Version:        %{go_version}
+Release:        %{baserelease}%{?dist}
+Summary:        The Go Programming Language
+# source tree includes several copies of Mark.Twain-Tom.Sawyer.txt under Public Domain
+License:        BSD and Public Domain
+URL:            https://go.dev
+Source0:        https://go.dev/dl/go%{go_source}.src.tar.gz
+# make possible to override default traceback level at build time by setting build tag rpm_crashtraceback
+Source1:        fedora.go
+
+# The compiler is written in Go. Needs go(1.4+) compiler for build.
+%if !%{golang_bootstrap}
+BuildRequires:  gcc-go >= 5
+%else
+BuildRequires:  golang > 1.4
+%endif
+%if 0%{?rhel} > 6 || 0%{?fedora} > 0
+BuildRequires:  hostname
+%else
+BuildRequires:  net-tools
+%endif
+# for tests
+BuildRequires:  pcre-devel, glibc-static, perl-interpreter, procps-ng
+
+Provides:       go = %{version}-%{release}
+
+# Bundled/Vendored provides generated by bundled-deps.sh based on the in tree module data
+# - in version filed substituted with . per versioning guidelines
+Provides: bundled(golang(github.com/google/pprof)) = 0.0.0.20211104044539.f987b9c94b31
+Provides: bundled(golang(github.com/ianlancetaylor/demangle)) = 0.0.0.20210905161508.09a460cdf81d
+Provides: bundled(golang(golang.org/x/arch)) = 0.0.0.20210923205945.b76863e36670
+Provides: bundled(golang(golang.org/x/crypto)) = 0.0.0.20211215153901.e495a2d5b3d3
+Provides: bundled(golang(golang.org/x/mod)) = 0.6.0.dev.0.20211102181907.3a5865c02020
+Provides: bundled(golang(golang.org/x/net)) = 0.0.0.20211209124913.491a49abca63
+Provides: bundled(golang(golang.org/x/sync)) = 0.0.0.20210220032951.036812b2e83c
+Provides: bundled(golang(golang.org/x/sys)) = 0.0.0.20211205182925.97ca703d548d
+Provides: bundled(golang(golang.org/x/term)) = 0.0.0.20210927222741.03fcf44c2211
+Provides: bundled(golang(golang.org/x/text)) = 0.3.8.0.20211105212822.18b340fc7af2
+Provides: bundled(golang(golang.org/x/tools)) = 0.1.9.0.20220329150752.294080fd2f5a
+Provides: bundled(golang(golang.org/x/xerrors)) = 0.0.0.20200804184101.5ec99f83aff1
+
+Requires:       %{name}-bin = %{version}-%{release}
+Requires:       %{name}-src = %{version}-%{release}
+Requires:       go-srpm-macros
+
+Patch1:       0001-Don-t-use-the-bundled-tzdata-at-runtime-except-for-t.patch
```

Comment:
we'll want to update this to include our patches as well, after de-duping 1 them